src/streaming/controllers/ScheduleController.js

```diff
@@ -470,6 +509,7 @@ function ScheduleController(config) {
         stop();
         fragmentController.detachModel(fragmentModel);
         isFragmentLoading = false;
+        completeRenderTimeCheck();
```

Comment:
This will cause an additional QUALITY_CHANGE_COMPLETE to be triggered when ScheduleController is reset which is misleading to anything subscribing to that event.

I would prefer if this didn't happen - I already have at least one application of this event in mind which will fail with the above behaviour.